static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -116,9 +114,8 @@ const setupPlayer = (
     onError
 ) => {
     const wantPersonalisedAds: boolean = consentState !== false;
-    const inPfpAdTargetingVariant: boolean = isInVariantSynchronous(
-        commercialYoutubePfpAdTargeting,
-        'variant'
+    const isPfpAdTargetingSwitchedOn: boolean = config.get(
```

Comment:
You can pass a  second [default value](https://github.com/guardian/frontend/blob/master/static/src/javascripts/lib/config.js#L10) argument to `config.get` which it will return if `switches.commercialYoutubePfpAdTargeting` doesn't exist on `window.guardian.config`, for example: 

```js
config.get('switches.commercialYoutubePfpAdTargeting', false)
```

Without this argument `config.get` will return `undefined` by default if `switches.commercialYoutubePfpAdTargeting` doesn't exist on `window.guardian.config`. 

In this instance the code as it stands will work fine if `undefined` is returned but the types indicate it's a `boolean`, which it won't necessarily be unless you add this 2nd argument.

Reply:
🤔 wonder why flow didn't complain then, pretty sure TS would have prompted me.

Reply:
hmm I guess because `config.get`'s return type is `any` you can to read all possible types from it.